.github/workflows/e2e-win-full-tests.yaml

```diff
@@ -19,8 +19,16 @@ jobs:
       fail-fast: true
       max-parallel: 1
       matrix:
-        # windows blocklist: "up-podman", "integration", "machineprovider", "proxyprovider", "up"
-        label: ["build", "ide", "machine", "provider", "ssh", "up-docker", "up-docker-compose", "up-docker-build", "up-docker-compose-build", "up-docker-wsl"]
+        label:
+          [
+            "build",
+            "ide",
+            "ssh",
+            "up-docker",
+            "up-docker-build",
+            "up-docker-compose",
+            "up-docker-wsl",
+          ]
```

Comment:
Would it make sense to change this to a YAML-style list?

Reply:
Makes sense, done :+1: